airflow/config_templates/default_airflow.cfg

```
@@ -59,6 +59,8 @@ fab_logging_level = WARN
 logging_config_class =
 
 # Log format
+colored_console_log = True
+colored_log_format = %%(blue)s[%%(asctime)s] {{%%(filename)s:%%(lineno)d}}%%(reset)s %%(log_color)s%%(levelname)s - %%(message)s%%(reset)s
```

```suggestion
colored_log_format = [%%(blue)s%%(asctime)s%%(reset)s] {{%%(blue)s%%(filename)s:%%(reset)s%%(lineno)d}} %%(log_color)s%%(levelname)s%%(reset)s - %%(log_color)s%%(message)s%%(reset)s
```

Is closer to your original screen shot (and I think, clearer) to read too.